cetmix_tower_aws/readme/CONFIGURE.md

```
@@ -0,0 +1,39 @@
+
+**Setting up AWS Access**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)